nav2_simple_commander/nav2_simple_commander/robot_navigator.py

```diff
@@ -307,11 +323,12 @@ def getResult(self):
             return TaskResult.CANCELED
         else:
             return TaskResult.UNKNOWN
-
+    
     def waitUntilNav2Active(self, navigator='bt_navigator', localizer='amcl'):
         """Block until the full navigation system is up and running."""
+        localizer = f'/{self.namespace}/amcl'
```

Comment:
This simply overrides the variable given, it should use the input. Can these defaults not have the self.namespace applied to them?

Reply:
``Can these defaults not have the self.namespace applied to them?``
I tried otherwise and did not work in my case. So, this is my theory,In my case my AMCL node also had the namespace and  the robot with the namespace needs to localize and it looks for the localization node associated with the namespace. Correct me If my approach is wrong.

Reply:
So if a User choose to use the AMCL node without namespace then I think there is no need to add ``localizer = f'/{self.namespace}/amcl'`` here